contracts/staking/Staking.sol

```diff
@@ -20,13 +21,15 @@ import "./libs/Stakes.sol";
  * Allocations on a Subgraph. It also allows Delegators to Delegate towards an Indexer. The
  * contract also has the slashing functionality.
  */
-contract Staking is StakingV2Storage, GraphUpgradeable, IStaking {
+contract Staking is StakingV2Storage, GraphUpgradeable, IStaking, Multicall {
     using SafeMath for uint256;
     using Stakes for Stakes.Indexer;
     using Rebates for Rebates.Pool;
 
     // 100% in parts per million
     uint32 private constant MAX_PPM = 1000000;
+    // Range to use for valid POI blocks from latest block number
+    uint256 private constant POI_VALID_BLOCK_RANGE = 256;
```

Comment:
This is only ~ an hour - given that the PoI needs to be for the first block in an epoch, does this mean that indexers will only be able to close allocations in the first hour of the epoch (or it won't be able to find the blockhash)?

Reply:
The Arbitration Charter needs to change to allow any indexer to be able to provide a proof within the N amount of blocks from the block they sent the transaction instead of using the start block of epoch.